cmd/dockerd/config_unix.go

```diff
@@ -45,9 +45,6 @@ func installConfigFlags(conf *config.Config, flags *pflag.FlagSet) error {
 	flags.StringVar(&conf.CgroupParent, "cgroup-parent", "", "Set parent cgroup for all containers")
 	flags.StringVar(&conf.RemappedRoot, "userns-remap", "", "User/Group setting for user namespaces")
 	flags.BoolVar(&conf.LiveRestoreEnabled, "live-restore", false, "Enable live restore of docker when containers are still running")
-	// TODO(thaJeztah): Used to produce a deprecation error; remove the flag and the OOMScoreAdjust field for the next release after v25.0.0.
```

Comment:
This comment suggests it should be dropped in v26.0, whereas the docs suggests it should be dropped in v25.0 (cf. https://docs.docker.com/engine/deprecated/#oom-score-adjust-for-the-daemon).

Reply:
Ah, yes, so;

- it was deprecated (warning only) in v24.0.0 https://github.com/moby/moby/pull/45315
- And changed to an _error_ for v25.0.0 https://github.com/moby/moby/pull/45484

So this PR should wait until after v25.0.0; the idea here is to produce a "more meaningful" error than

```bash
Status: unknown flag: --oom-score-adjust
See 'dockerd --help'., Code: 125
```

So I guess we should put this PR on the "future" milestone, and keep it  in draft until we started a "25.0" release branch